.github/pull_request_template.md

```diff
@@ -26,10 +26,12 @@ Not applicable.
 
 
 
-# Checklist
-
-- [ ] l10n changes have been submitted to the l10n repository, if any.
+# Checklist (Definition of Done)
+- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
 - [ ] All acceptance criteria are met.
-- [ ] I've added or updated relevant docs in the docs/ directory.
+- [ ] User Story has been updated (if needed) to match changes made during the development process.
+- [ ] I've added or updated relevant docs in the docs/ directory
+- [ ] User Story has been updated (if needed) with suggestions for QA when this PR is deployed to stage.
```

Comment:
do we have to say "User Story" or can we say "ticket"? I'm thinking some of these check-boxes apply to bugs too? (but I guess this is the "When adding a new feature" section anyway?

Reply:
How about Jira ticket instead of User Story?

Reply:
Sounds good!

Reply:
I've updated it. Let me know if this works.